### DISCUSSION TITLE
Weird Dashboard

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#5678] [Ideas] [answered]
Hello all!

It would be great if in the network dataset in kibana if a weird event.dataset option was available.  Trying to get to the information about the weird (zeek) traffic is difficult.

**Comment by dougburks** [comment, marked as answer]
Have you tried the following options?

- On our main Kibana dashboard, click Network. On the Network dashboard, go to the Dataset visualization, and click `weird` or click the `+` sign to the right of it:
![image](https://user-images.githubusercontent.com/1659467/135163063-8da62c65-b9e5-40ce-a164-a976e6b75774.png)

OR

- Search Kibana for `event.dataset.keyword: weird`:
![image](https://user-images.githubusercontent.com/1659467/135163107-8f614af1-dc61-4ca0-a958-546a3f9151e5.png)

OR

- Our Hunt interface already includes a default dashboard for Weird:
![image](https://user-images.githubusercontent.com/1659467/135162485-6bfa3466-d61f-4207-897a-9fc4d46610e6.png)

**Reply by oneCrazyAdmin** [reply]
that make waaayyy too much sense. LOL

Thank you!
